code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -263,8 +263,33 @@
 	desc = "When the station falls into chaos, it's nice to have a friend by your side."
 	attack_verb = list("chopped", "sliced", "cut")
 	hitsound = 'sound/weapons/rapierhit.ogg'
+	force = 12
+	can_be_hit = TRUE // be a shit and you can get your ass beat
+	max_integrity = 100
+	obj_integrity = 100
 	var/possessed = FALSE
 
+/obj/item/nullrod/scythe/talking/proc/debug_spawn()
```

Comment:
Going to guess you probably want to remove this

Reply:
I *kinda* wanna leave it in for further testing just because of how much of a bitch it is rn, although it can be removed if that's a demand

Reply:
Fair enough, I don't think it *needs* to go, especially if it keeps things easier to work with. Just might be a demand before merge.

Reply:
this should be removed before merge unless it's converted into an actual usable tool (which probably won't be used tbqh). Stuff shouldn't be left in that only codermins can access with magic.